vaccine_feed_ingest/cli.py

```diff
@@ -82,8 +82,9 @@ def available_sites(state: Optional[str]) -> None:
     """Print list of available sites, optionally filtered by state"""
 
     for site_dir in site.get_site_dirs_for_state(state):
-        has_fetch = bool(site.find_executeable(site_dir, common.PipelineStage.FETCH))
-        has_parse = bool(site.find_executeable(site_dir, common.PipelineStage.PARSE))
+        is_arcgis_site = site_dir.name.endswith('arcgis')
```

Comment:
Some non-ArcGIS sites, [like `ok/vaccinate_gov`](https://github.com/CAVaccineInventory/vaccine-feed-ingest/tree/main/vaccine_feed_ingest/runners/ok/vaccinate_gov), use a `.yml` file as well.

Instead of filtering for only sites which end in `arcgis`, could we capture the broader use case? The [logic for finding files used when running the commands](https://github.com/CAVaccineInventory/vaccine-feed-ingest/blob/d3fb58da986fbda52f01a6750201e2e4febcf791/vaccine_feed_ingest/stages/ingest.py#L91-L94) might provide a good template to follow.

Reply:
I think I captured the broader use case like you suggested, please take a look!